lib/puppet/application/cert.rb

```diff
@@ -145,7 +164,10 @@ def help
   needs to be restarted after revoking certificates.
 
 * sign:
-  Sign an outstanding certificate request.
+  Sign an outstanding certificate request. If '--interactive' or '-i' is
+  supplied the user will be prompted to confirm that they are signing the
+  correct certificate (recommended). If '--assume-yes' or '-y' is supplied
```

Comment:
Was `-f` and `--force` considered instead?  As a Git user that would feel much more familiar to me, and for some reason "assume yes" just seems strange.